src/api/api.cc

```diff
@@ -11282,6 +11283,24 @@ extern "C" DLLEXPORT void V8RecordReplayWarning(const char* format,
   }
 }
 
+bool recordreplay::HadMismatch() {
+  if (IsRecordingOrReplaying()) {
+    // It is an error to call this during record time.
+    CHECK(!recordreplay::IsRecording());
+    return gRecordReplayHadMismatch();
+  }
+  return false;
+}
+
+bool V8RecordReplayHadMismatch() {
```

Comment:
Should this also get extern "C" linkage?

Reply:
Fixed.